journal/boyington.tex

```diff
@@ -0,0 +1,878 @@
+It is assumed because to flux-weight the cross section requires {\it a priori} knowledge of the true flux spectrum, the obtention of which is the goal of our work here.
+Deconvolution refers to the class of methods used to obtain the flux vector.
+
+Many different methods of deconvolution have been studied and shown to have varying levels of largely application-dependent success.
+Two popular methods used in similar applications to this are included in the RSICC Peripheral Science Routine Collection, namely, MAXED and Gravel.
+
+MAXED, arguably the more sophisticated of the two finds its basis in information theory and leverages the principle of maximum entropy \cite{reginatto1999maxed}.
+It seeks to solve a system of equations comprised of \EQ{eq:response} and \EQ{eq:omega} while maximizing \EQ{eq:entropy}
+
+\begin{equation}
+  \sum \frac{\epsilon_i^2}{\sigma_k^2} = \Omega
+\label{eq:omega}
+\end{equation}
+
+\begin{equation}
+  S = - \sum [f_i \ln(f_i/f_i^{DEF}) + f_i^{DEF} - f_i]
+\label{eq:entropy}
+\end{equation}
+
+In the above equations, $\Omega$ is a user-defined parameter that relates the unknown error $\epsilon_i$ to an assumed variance in response $\sigma_k^2$.
+$S$ is the entropy we are trying to maximize, and $f_i^{DEF}$ is the default starting spectrum.
+This maximization of $S$ is done through use of simulated annealing and detailed in Reginatto and Goldhagen \cite{reginatto1999maxed}.
+The closed form solution allows for the propogation of uncertainty, a feature not available in other, iterative-type deconvolution methods.
+
+Gravel employs a modified version of the Sand II algorithm which was first detailed in a 1967 report from the Air Force Weapons Laboratory \cite{mcelroy1967computer}.
+The following equation describes the (relatively simple) algorithm:
+
+\begin{equation}
+ f_i^{J+1} = f_i^{J} \exp(\frac{\sum_k W_{ik}^J \log(\frac{N_k}{\sum_{i'} R_{ki'}f_{i'}^J})}{\sum_k W_{ik}^J}) 
+\label{eq:sand}
+\end{equation}
+
+where $N_k$ is the measured response, $\sigma_k$ is the measurement error, $R_{ki}$ is the response matrix, and $W_{ik}^J$ is a weighting term:
+
+\begin{equation}
+ W_{ik}^J = \frac{R_{ki} f_i^J}{\sum_{i'} f_{i'}^J} \frac{N_k^2}{\sigma_k^2} 
+\label{eq:weighting}
+\end{equation}
+
+$J$ represents the current iteration, $i$ is the energy group index, and $k$ is the response index.
+Because there is no closed form solution to the algorithm, uncertainty propagation is not possible here \cite{reginatto2004umg}.
+
+
+If we further allow the assumption that $\epsilon_i$ is negligible, this allows for the utilization of three other methods.
+One of these is to compute the minimum-norm solution, i.e.,
+
+\begin{equation}
+  \bm{\phi} =  \bm{\Sigma}  [\bm{\Sigma} \bm{\Sigma}^T]^{-1}\mathbf{r} \, ,
+\end{equation}
+
+which yields a spectrum $\bm{\phi}$ having the smallest Euclidean norm $|\bm{\phi}|_2 = \sqrt{\bm{\phi}^T\bm{\phi}}$.
```

Comment:
Make sure not to have a space between equations and a following line (or use `\noindent`)